Project.toml

```diff
@@ -4,19 +4,25 @@ authors = ["Rachel Kurchin <rkurchin@cmu.edu>", "Sean Sun"]
 version = "0.1.0"
 
 [deps]
+BenchmarkTools = "6e4b80f9-dd63-53aa-95a3-0cdb28fa8baf"
```

Comment:
Is this required for run any tests? Otherwise I think it's better to just use this for testing rather than adding this to the package dependencies.

Reply:
oh good call I'll take it out

Reply:
oh wait, I already did...